### PR TITLE
Resolved Cors Issue

### PR DESCRIPTION
**Pull request by zeeshan595**

### CODE REVIEW

**Review by JettTech — CHANGES_REQUESTED** [review]
I'm concerned this change will break develop.  Can we have a conversation about this first?

File: overlays/holo-nixpkgs/membrane-proof-service/src/app.js [hunk 1]
```diff
@@ -47,7 +47,6 @@ app.get("/verify-registration-code/", async (req, res) => {
     res.set("Access-Control-Allow-Origin", "*")
   } else {
     res.set("Access-Control-Allow-Origin", "https://quickstart.holo.host")
-    res.set("Access-Control-Allow-Origin", "https://devnet-quickstart.holo.host")
```
[the remark this answers is not in the document]
Comment (JettTech) [reply overlays/holo-nixpkgs/membrane-proof-service/src/app.js:50]: Is the develop membrane proof server setup to run in test mode?  Currently, I only see the script that runs it in production mode.

[the remark this answers is not in the document]
Comment (JettTech) [reply overlays/holo-nixpkgs/membrane-proof-service/src/app.js:50]: @peeech - I understand the intent of this pr to be the fix for issue #1291.  As it is a simple update, my vote would be to just do the lasting fix here in this pr. :+1:

[the remark this answers is not in the document]
Comment (peeech) [reply overlays/holo-nixpkgs/membrane-proof-service/src/app.js:50]: I'll let you go ahead and do it. I'm in the middle of HC immersive, not able to spend time in front of the laptop

**Pull request closed without merging**